ironfish-cli/src/commands/accounts/balance.ts

```diff
@@ -41,32 +46,51 @@ export class BalanceCommand extends IronfishCommand {
   async start(): Promise<void> {
     const { flags, args } = await this.parse(BalanceCommand)
     const account = args.account as string | undefined
+    const assetIdentifier = flags.assetIdentifier ?? Asset.nativeIdentifier().toString('hex')
```

Comment:
This should be populated by the backend, not the front end. It should respect the balance from the node, node the CLI.

Reply:
Fixed